### ISSUE TITLE
Warning: Specter DIY failed to establish socket connection. Error: [Errno 111] Connection refused

### ISSUE THREAD

**Issue by KD134993** [cryptoadvance/specter-desktop#1384]
I've got a few issues with Specter that I'm trying to fix but hitting brick walls. 

1. I get this error when I run Specter using the following command: python3 -m cryptoadvance.specter server 

"Warning: Specter DIY failed to establish socket connection. Error: [Errno 111] Connection refused"

Specter starts and initially ran on http://127.0.0.1:25441. I then setup a password for admin and now it won't remember the password. When I try to login I'm locked out. And yes, it's definitely the correct password.

I edited the config.json file last night and changed the password method to ''none", cleared my browser history and managed to get back to the standard opening page of specter. However, today I'm locked out again and the method I tried last night of resetting the password method to "none" isn't working anymore and I'm currently locked out and can't get back in. 

2. I was also having issues with my Trezor last night when I tried to connect. I installed and configured the USB packages and Specter could identify the hardware. However, after I got passed the login pin on my Trezor, it just stalled and I couldn't get any further with it using the software.

I'm currently running the latest Trezor firmware version 1.10.2
I've ran the upgrade command pip3 install cryptoadvance.specter --upgrade
I'm on Linux Ubuntu 20.04
I'm running bitcoin deamon in the background with Tor enabled. 

Thanks.

**Comment by KD134993** [comment]
Quick update. I restarted python3 -m cryptoadvance.specter server and cleared my cache and now I'm back at the welcome page of specter. However, I have no doubt the password and Trezor issue is still there. I need to crack on with the ministry of node setup so I'll wait for some feedback before I do any further work on this app.

**Comment by KD134993** [comment]
Same issue when I try to connect to --host:

python3 -m cryptoadvance.specter server --host 0.0.0.0
Initializing HWI...
Warning: Specter DIY failed to establish socket connection. Error: [Errno 111] Connection refused

**Comment by KD134993** [comment]
UPDATE. I've managed to resolve a lot of these issues. 

1. I now have Trezor functioning. 
2. I can now login to Specter
3. I have connected to Tor and can access Spector through a .onion

The only issue that remains is:

I get this error when I run Specter using the following command: python3 -m cryptoadvance.specter server

"Warning: Specter DIY failed to establish socket connection. Error: [Errno 111] Connection refused"
